### DISCUSSION TITLE
[Feature] <Preserve transcoded metadata>

### DISCUSSION THREAD

**Discussion by lynrayy** [immich-app/immich#6990] [Feature Request]
### The feature

Add option to preserve video metadata after transcoding. This will save parity between immich gallery and file system and save sorting by date.
Most important metadata is created date and edited date. Other data is appreciated 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by lynrayy** [comment]
Personally, I perceive transcoding as a way to compress unoptimized videos shot on camera, downloaded from the Internet, etc. (a way to save space and traffic). Expected behavior - the video goes to the server and is optimized, saving all data. But instead, at the moment the video loses its meta data and ends up in the most recent (in Explorer), and is not located chronologically. I assume that if you download it from the gallery back to your phone, the chronological order will also be disrupted. Since you have a different vision of this, I propose making this function optional, otherwise I personally don’t see the point of using transcoding - there is more harm than good.

**Reply by bo0tzz** [reply]
The original files are all kept. The transcoded files are separate and exist just for performance and compatibility. If you're browsing the files in explorer, you probably want to ignore the encoded-video folder.

**Reply by lynrayy** [reply]
Huh? But i have video with reset metadata in library/admin folder. 🤔

Anyway, this feature needs either a proper explanation (I thought that it transcodes original files), or remake to function like the one I described - recode/compress original files with preserving metadata

**Reply by lynrayy** [reply]
>  exist just for performance and compatibility

In this case i don't need it at all
